internal/webhook/external/media_types.go

```diff
@@ -0,0 +1,36 @@
+package external
+
+import (
+	"slices"
+
+	"github.com/akuity/kargo/internal/helm"
+	"github.com/akuity/kargo/internal/image"
+)
+
+var containerImageMediaTypes = []string{
+	"application/vnd.oci.image.index.v1+json",
+	"application/vnd.oci.image.manifest.v1+json",
+	"application/vnd.docker.distribution.manifest.list.v2+json",
+	"application/vnd.docker.distribution.manifest.v2+json",
+}
+
+func isContainerImageMediaType(mediaType string) bool {
+	return slices.Contains(containerImageMediaTypes, mediaType)
+}
+
+func isHelmChartMediaType(mediaType string) bool {
+	return mediaType == "application/vnd.cncf.helm.config.v1+json"
```

Comment:
This could be a `const`?

Reply:
Fixed in c4e2c2fb73ad04d354b18e3a46b9968547fa1b6d